application/src/main/java/org/togetherjava/tjbot/features/moderation/TransferQuestionCommand.java

```diff
@@ -96,15 +96,18 @@ public void onMessageContext(MessageContextInteractionEvent event) {
         String originalChannelId = event.getTarget().getChannel().getId();
         String authorId = event.getTarget().getAuthor().getId();
         String mostCommonTag = tags.getFirst();
+        final int TITLE_MAX_LENGTH = 50;
         String chatGptPrompt =
                 "Summarize the following text into a concise title or heading not more than 4-5 words, remove quotations if any: %s"
                     .formatted(originalMessage);
         Optional<String> chatGptResponse = chatGptService.ask(chatGptPrompt, "");
         String title = chatGptResponse.orElse(createTitle(originalMessage));
+        title = title.replaceAll("^\"|\"$", "").replaceAll("^'|'$", "");
```

Comment:
```suggestion
        title = title.replaceAll("^['\"]|['\"]$", "");
```
we can maybe merge the two replaceAll calls like this

Reply:
do we want that though? the logic is kinda different now.
Imagine a title like this: `"!!" in 'Kotlin'` or similar